### PR TITLE
Fix encryption sample comments to be more clear

### PR DESCRIPTION
**Pull request by longquanzheng**
## What was changed

Fix encryption sample comments to be more clear

## Why?

The comments has confusing me and turns out it's not accurate. 

## Checklist

1. Closes 
None

2. How was this tested:

No test needed.

3. Any docs updates needed?

No need to update

### CODE REVIEW

File: encryption/starter/main.go [hunk 1]
```diff
@@ -14,21 +14,20 @@ import (
 func main() {
 	// The client is a heavyweight object that should be created once per process.
 	c, err := client.Dial(client.Options{
-		// If you intend to use the same encryption key for all workflows you can
-		// set the KeyID for the encryption encoder like so:
-		//
-		// Set DataConverter to ensure that workflow inputs and results are
-		// encrypted/decrypted as required.
+		// If you intend to let the dataConverter to decide encryption key for all workflows
+		//you can set the KeyID for the encryption encoder like so:
```
Comment (cretz) [inline encryption/starter/main.go:18]:
```suggestion
		// you can set the KeyID for the encryption encoder like so:
```

File: encryption/starter/main.go [hunk 2]
```diff
@@ -14,21 +14,20 @@ import (
...
 		//
 		//   DataConverter: encryption.NewEncryptionDataConverter(
 		// 	  converter.GetDefaultDataConverter(),
 		// 	  encryption.DataConverterOptions{KeyID: "test", Compress: true},
 		//   ),
 		//
 		// In this case you do not need to use a ContextPropagator.
+		// You also can implement the EncryptionDataConverter to decide the encryption key
```
Comment (cretz) [inline encryption/starter/main.go:26]:
```suggestion
		// You also can implement the data converter to decide the encryption key
```
What is `EncryptionDataConverter` referring to?

Comment (longquanzheng) [reply encryption/starter/main.go:26]: In `data_converter.go` the constructor name is `NewEncryptionDataConverter`. I thought that's what you want to call it to be different from regular dataConverter.

Comment (cretz) [reply encryption/starter/main.go:26]: No need for proper name

File: encryption/worker/main.go [hunk 3]
```diff
@@ -13,21 +13,20 @@ import (
 func main() {
 	// The client and worker are heavyweight objects that should be created once per process.
 	c, err := client.Dial(client.Options{
-		// If you intend to use the same encryption key for all workflows you can
-		// set the KeyID for the encryption encoder like so:
-		//
-		// Set DataConverter to ensure that workflow inputs and results are
-		// encrypted/decrypted as required.
+		// If you intend to let the dataConverter to decide encryption key for all workflows
+		//you can set the KeyID for the encryption encoder like so:
```
Comment (cretz) [inline encryption/worker/main.go:17]:
```suggestion
		// you can set the KeyID for the encryption encoder like so:
```

Missed this one.

File: encryption/starter/main.go (hunk 2 above)
Comment (longquanzheng) [reply encryption/starter/main.go]: that's fine, I changed to dataConverter

**Comment by cretz** [comment]
@longquanzheng - Sorry, you're good! Let me update with the main branch then I'll merge.

**Review by cretz — APPROVED** [review]

**Pull request merged**